### PR TITLE
Issue 12

### PR DESCRIPTION
**Pull request by sebastianrakel**
Script Parameter implementation

### CODE REVIEW

**Comment by bastelfreak** [comment]
this will implement https://github.com/virtapi/marmoset/issues/12

**Comment by bastelfreak** [comment]
ignore travis, we do not have a .travis.yml , this will come in https://github.com/virtapi/marmoset/issues/15

File: marmoset/pxe/client_config.py [hunk 1]
```diff
@@ -119,8 +139,8 @@ def __write_config_file(self, content, path=None):
 
     def __expand_template(self, label, options = None):
         '''Return the config file content expanded with the given values.'''
-        if options is None:
-            options = ''
+        options = " ".join(options)
```
Comment (aibor) [inline marmoset/pxe/client_config.py:142]: This will throw an `TypeError` exception, if options is None

File: marmoset/pxe/client_config.py [hunk 2]
```diff
@@ -69,19 +73,35 @@ def get_label(self):
...
+        '''Parse the script option form the config file.'''
+        with open(self.file_path()) as f:
+            for line in f:
+                m = re.match(' *APPEND.*script=(\S+)', line)
+                if m is not None:
+                    return m.group(1)
+
 
     def create(self, pxe_label):
+        options = []
```
Comment (aibor) [inline marmoset/pxe/client_config.py:86]: This line breaks the docstring.

**Comment by sebastianrakel** [comment]
Will fix issues mentioned from @aibo soon

**Pull request merged**